packages/nextra/src/client/components/collapse.tsx

```diff
@@ -60,7 +60,11 @@ export function Collapse({
   useEffect(() => {
     initialRender.current = false
   }, [])
-
+  // Add inner <div> only if children.length != 1
+  const newChildren = useMemo(
+    () => (Children.count(children) === 1 ? children : <div>{children}</div>),
+    [children]
+  )
```

Comment:
this to use is for 1 inner div less in
<img width="1060" alt="image" src="https://github.com/user-attachments/assets/9f430141-04bf-4d8c-a972-8dd5c710bb38">
and
<img width="1035" alt="image" src="https://github.com/user-attachments/assets/6e54f3a6-523a-414b-b946-820e9812594b">

Reply:
hm, I guess it produces `Uncaught Error: Minified React error #425` 🧐
<img width="1628" alt="image" src="https://github.com/user-attachments/assets/624f43e2-1b34-4cbd-adad-1dc52f582841">